components/ExerciseCard.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { View, StyleSheet, Text, TextInput } from "react-native";
+import { Button, Icon } from "react-native-elements";
+import WorkoutCard from "../components/WorkoutCard";
+import { FontStyles } from "../styles/global";
+import SetDetailsView from "../screens/SetDetailsView";
+import { isTSExpressionWithTypeArguments } from "@babel/types";
+
+class ExerciseCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: 0,
+      name: "",
+      description: "",
+      exerciseSets: [
+      ]
+    };
+
+    this.addSet = this.addSet.bind(this);
+    this.changeName = this.changeName.bind(this);
+    this.changeDesc = this.changeDesc.bind(this);
+  }
+
+  addSet() {
+      let set ={
+        id: 0,
```

Comment:
Not sure if we should id's manually or firebase generates something unique. 
Can fix it later I think